rasa/core/channels/mattermost.py

```diff
@@ -39,6 +49,34 @@ def __init__(self, url, team, user, pw, bot_channel):
         json_message.setdefault("message", "")
         self.post("/posts", json_message)
 
+    async def send_text_with_buttons(
+        self,
+        recipient_id: Text,
+        text: Text,
+        buttons: List[Dict[Text, Any]],
+        **kwargs: Any
+    ) -> None:
+        """Sends buttons to the output."""
+
+        # buttons is a list of tuples: [(option_name, payload)]
```

Comment:
Not sure about the tuples. It seems more like a list of objects to me.

Reply:
True, I took this from the Mattermost documentation, seems to be incorrect.